Jenkinsfile

```
@@ -546,6 +546,15 @@ pipeline {
             }
           }
         }
+        stage('Publish Docker image') {
```

Set the env variables with the docker specific details:

```suggestion
        stage('Publish Docker image') {
          environment {
            DOCKER_SECRET = 'secret/apm-team/ci/docker-registry/prod'
            DOCKER_REGISTRY = 'docker.elastic.co'
```